### ISSUE TITLE
Jellyfin Media Player 1.6.1 randomly stopping playrback/jumping to next episode

### ISSUE THREAD

**Issue by Jinxyface** [jellyfin/jellyfin-media-player#140]
Four considerations:
 - Please do not open bug reports to ask questions. Use the Discussions feature instead.
 - Please make sure the issue only pertains to Jellyfin Media Player. If it also occurs in the web client, send the issue to jellyfin-web instead.
 - Please make sure that your issue is not being caused by errors in custom CSS or note that you are using custom CSS.
     - Notably, there have been instances of custom CSS breaking TV mode.
     - You can disable custom CSS under Display in the user settings.
 - Please provide logs. You can drag the log file into the issue to attach it.
     - Windows: `%LOCALAPPDATA%\JellyfinMediaPlayer\logs`
     - Linux: `~/.local/share/jellyfinmediaplayer/logs/`
     - Linux (Flatpak): `~/.var/app/com.github.iwalton3.jellyfin-media-player/data/jellyfinmediaplayer/logs/`
     - macOS: `~/Library/Logs/Jellyfin Media Player/`

**Describe the bug**
When playing an episode in Jellyfin Media Player, sometimes it will randomly skip to the next episode. This does not happen when watching Jellyfin through a browser, only in Jellyfin Media Player (and it's random, not very consistent to reproduce)

**To Reproduce**
Start Jellyfin Media Player
Play episode
Sometimes (around halfway in), the program will randomly jump to the next episode without finishing the currently playing file

**Expected behavior**
Jellyfin Media Player will only move to the next episode once the current file is done being played

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 1.6.1
 - Installation Method: Windows Installer

**Additional context**
Logs attached, this is the latest part of the log after the file has been skipped over:

- [2021-09-06 19:39:31.535 -05:00] [INF] [86] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Media Player" "1.6.1" playing "Big Stan on Campus". Stopped at "861027" ms
- [2021-09-06 19:39:31.558 -05:00] [INF] [54] Trakt.ServerMediator: Playback Stopped
- [2021-09-06 19:39:31.559 -05:00] [INF] [54] Trakt.ServerMediator: Item Not fully played. Tell trakt.tv we are no longer watching but don't scrobble
- [2021-09-06 19:39:31.697 -05:00] [INF] [63] Jellyfin.Api.Controllers.MediaInfoController: GetPostedPlaybackInfo profile: DeviceProfile { Name: "Jellyfin Media Player", Id: null, Identification: null, FriendlyName: null, Manufacturer: null, ManufacturerUrl: null, ModelName: null, ModelDescription: null, ModelNumber: null, ModelUrl: null, SerialNumber: null, EnableAlbumArtInDidl: False, EnableSingleAlbumArtLimit: False, EnableSingleSubtitleLimit: False, SupportedMediaTypes: "Audio,Photo,Video", UserId: null, AlbumArtPn: null, MaxAlbumArtWidth: 0, MaxAlbumArtHeight: 0, MaxIconWidth: null, MaxIconHeight: null, MaxStreamingBitrate: 8000000, MaxStaticBitrate: 8000000, MusicStreamingTranscodingBitrate: 1280000, MaxStaticMusicBitrate: null, SonyAggregationFlags: null, ProtocolInfo: null, TimelineOffsetSeconds: 5, RequiresPlainVideoItems: False, RequiresPlainFolders: False, EnableMSMediaReceiverRegistrar: False, IgnoreTranscodeByteRangeRequests: False, XmlRootAttributes: [], DirectPlayProfiles: [DirectPlayProfile { Container: null, AudioCodec: null, VideoCodec: null, Type: Video }, DirectPlayProfile { Container: null, AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: null, AudioCodec: null, VideoCodec: null, Type: Photo }], TranscodingProfiles: [TranscodingProfile { Container: null, Type: Audio, VideoCodec: null, AudioCodec: null, Protocol: null, EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "ts", Type: Video, VideoCodec: "h264,h265,hevc,mpeg4,mpeg2video", AudioCodec: "aac,mp3,ac3,opus,flac,vorbis", Protocol: "hls", EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: "6", MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "jpeg", Type: Photo, VideoCodec: null, AudioCodec: null, Protocol: null, EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }], ContainerProfiles: [], CodecProfiles: [], ResponseProfiles: [], SubtitleProfiles: [SubtitleProfile { Format: "srt", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "srt", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ass", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ass", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "sub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "sub", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ssa", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ssa", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "smi", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "smi", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "pgssub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "dvdsub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "pgs", Method: Embed, DidlMode: null, Language: null, Container: null }] }
- [2021-09-06 19:39:31.697 -05:00] [INF] [54] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Media Player" "1.6.1" playing "Big Stan on Campus". Stopped at "0" ms
- [2021-09-06 19:39:31.738 -05:00] [INF] [63] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "Jinxyface". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
- [2021-09-06 19:39:31.739 -05:00] [INF] [63] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Jellyfin Media Player", Path: "P:\TV Shows\American Dad\Season 12\American Dad! - 12x05 - Now and Gwen.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
- [2021-09-06 19:39:31.740 -05:00] [INF] [63] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Jellyfin Media Player", Path: "P:\TV Shows\American Dad\Season 12\American Dad! - 12x05 - Now and Gwen.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
- [2021-09-06 19:39:31.741 -05:00] [INF] [63] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Jellyfin Media Player", Path: "P:\TV Shows\American Dad\Season 12\American Dad! - 12x05 - Now and Gwen.mkv", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
- [2021-09-06 19:39:31.759 -05:00] [INF] [87] Trakt.ServerMediator: Playback Stopped
- [2021-09-06 19:39:31.759 -05:00] [INF] [87] Trakt.ServerMediator: Item Not fully played. Tell trakt.tv we are no longer watching but don't scrobble
- [2021-09-06 19:39:31.788 -05:00] [INF] [73] Trakt.ServerMediator: Playback Started
- [2021-09-06 19:39:36.399 -05:00] [INF] [88] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "68.21.155.219" closed

[log_20210906.log](https://github.com/jellyfin/jellyfin-media-player/files/7118212/log_20210906.log)

**Comment by marcusds** [comment]
I am also having this problem on Jellyfon 10.7.7 while watching via Chromecast.

**Comment by Mettchen** [comment]
Most likely related to https://github.com/jellyfin/jellyfin/issues/7296

Should be fixed in 10.8 - you can try beta testing it

**Comment by Mustachipleb** [comment]
Updated to 10.8 a few days ago and consistently encountering the same problem, it does not seem to be fixed.

**Comment by k6Q28k5w5eWuRuuzTg9j7GrKCbjt7** [comment]
On 10.8.0 (and now on 10.8.1), player cant jump to next (hangs). Same if done by controls or if let the video finish fully. Have to hard kill the player then play the file directly on at a time. 

There was no problems pre 10.8, this occured post 10.8. 

 file:///home/valk/player.log

**Comment by Mettchen** [comment]
> On 10.8.1 running the flatpak version of the player (latest). Player can't jump to the next (by controls) resulting in nonresponsive "crash". Same occurs if I let the video finish and player tries to play the next in the playlist. I have to hard kill the player then play the file directly on at a time. This only occurs on the flatpak version of the media player, not browser.
> 
> Note that this didnt occur pre 10.8, only post updating to 10.8.
> 
> [player.log](https://github.com/jellyfin/jellyfin-media-player/files/9224091/player.log)

Seems like an unrelated problem to this issue, you should create a new issue if you want any chance of somebody looking at it.

**Comment by iwalton3** [comment]
I recall Cloudflare has caused issues in the past, but it looks like you already tried the remedy from then: https://github.com/jellyfin/jellyfin-media-player/issues/174#issuecomment-1084038050

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
